tests/testthat/test-dec.R

```diff
@@ -22,6 +22,13 @@ s <- httpuv::startServer(conf$host, port = conf$port,
 tarfile <- enc(filename = content_file, pubkey_holder = "local", pid = pid)
```

Comment:
Her er `pubkey_holder`-argumentet `"local"`, men i funksjonen skal det vera `"file"`. Viss dette «fungerer», er altså noko gale …

Reply:
Fra dokumentasjonen av funksjonen:
> @param pubkey_holder string defining the provider of the public key used for
#' encryption of the symmetric key. Currently, 'github' is the only valid
#' pubkey holder. If a local pubkey is to be used (see parameter \code{pubkey},
#' \code{pubkey_holder} may be set to NULL or some other value.

Jeg er klar over at "koden" brukt i enhetstestene kan være dårlig strukturert og uklar. Der har jeg nok et betydelig forbedringspotensiale...

Ellers å er denne delen av pakken (det å kunne sende filer) ikke ferdig utbygd (av årsaker som jeg ikke bruker plass på her). Bla derav v0.x.

Reply:
Men sjølve *koden* for funksjonen har denne linja:

    if (pubkey_holder == "file") {

Altså, `"file"` og ikkje `"local"`. Og det er vel nettopp denne delen av koden som testen er meint å testa, eller?

Reply:
Jeg skal se litt nærmere på dette når jeg oppdaterer koden etter dine innspill. Kommer tilbake til denne, da.

Reply:
Denne gjelder enhetstesting av *dekryptering* og det er således ikke `pubkey_holder == "file"` (som brukes ved kryptering) som skal testes.

Når det er sagt så er faktisk ikke bruk av `pubkey_holder == "file"` ved kryptering testet og årsaken er bla som nevnt over at denne delen av pakken ikke er ferdig utbygd. Dette kommer jeg tilbake til i en senere versjon.